src/instance/discord/discord-instance.ts

```diff
@@ -158,6 +167,18 @@ export default class DiscordInstance extends ClientInstance<DiscordConfig> {
     await this.sendEmbed(channels, event.removeLater, embed)
   }
 
+  private async onProfanityWarning(event: ProfanityWarningEvent): Promise<void> {
+    const embed = {
+      description: `Message by ${escapeDiscord(event.username)} filtered:\n\n'${escapeDiscord(event.oldMessage)}'\n->\n'${escapeDiscord(event.newMessage)}'`,
+      color: 0x8a_2d_00,
+      footer: {
+        text: `${event.instanceName} in ${event.channelType} chat`
+      }
```

Comment:
These are custom. There is a color palette for discord messages you can use. I think it is an enum called `ColorScheme`. Same with footer.

Reply:
I found the colors enum.
Any way you could specify what needs to change with the footer? It seems to me like the current implementation will give the most information, as moderators need to know where the profanity flagging took place.